soc/xtensa/intel_adsp/ace_v1x/include/ace_v1x-regs.h

```diff
@@ -162,6 +162,41 @@ struct ace_intc {
 
 #define ACE_INTC_IRQ DT_IRQN(DT_NODELABEL(ace_intc))
 
+/* Power control */
+#define PWRCTL_REG 0x71b90
```

Comment:
Raw register addresses are discouraged.  This should live in a register block defined via devicetree (either the struct scheme or a more traditional base address macro would be fine)

Reply:
It looks like this file will need to be reworked in general -- there are other definitions of the register addresses similar to this.

Reply:
Agree, this will be done.

Reply:
`git grep "#define" | grep 0x  | wc -l
18340
`